<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: #121212</string>
   </property>
   <widget class="QLabel" name="Articule">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>2000</width>
      <height>1000000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;
border: 2px solid #D4A017;
border-radius: 25px;
color: #FFD700;
font-size: 24px;
font-family: Roboto;
font-weight: 500</string>
    </property>
    <property name="text">
     <string>E   X   T   R   E   M   U   M     F   I   N   D   E   R</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Graph">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>380</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400000</width>
      <height>100000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;
border: 2px solid #D4A017;
border-radius: 15px;
color: #FFD700;
padding: 15px;
font-size: 18px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="FuncInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;
border: 2px solid #D4A017;
border-radius: 15px;
color: #FFD700;
font-size: 18px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Введите функцию. Например: x ** 3</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="Interval">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;
border: 2px solid #D4A017;
border-radius: 15px;
color: #FFD700;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Interval1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1A1A1A;
color: #FFD700;
border: 0px solid red;
font-size: 18px;
font-family: Roboto;
margin-right: 20px</string>
     </property>
     <property name="text">
      <string>Интервал: [ </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LeftIntervalInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: #FFD700;
font-size: 16px;
border-radius: 0px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RightIntervalInput">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: #FFD700;
font-size: 16px;
border-radius: 0px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Interval3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1A1A1A;
color: #FFD700;
border: 0px solid red;
font-size: 18px;
font-family: Roboto</string>
     </property>
     <property name="text">
      <string>]</string>
     </property>
    </widget>
    <widget class="QLabel" name="Interval2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1A1A1A;
color: #FFD700;
border: 0px solid red;
font-size: 18px;
font-family: Roboto</string>
     </property>
     <property name="text">
      <string>;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Epsilon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;
border: 2px solid #D4A017;
border-radius: 15px;
color: #FFD700;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Epsilon1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1A1A1A;
color: #FFD700;
border: 0px solid red;
font-size: 18px;
font-family: Roboto</string>
     </property>
     <property name="text">
      <string>Допустимая погрешность:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EpsilonInput">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: #FFD700;
font-size: 16px;
border-radius: 0px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="StartEngine">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background: #1A1A1A;
	border: 2px solid #D4A017;
	border-radius: 10px;
	color: #FFD700;
	font-size: 24px;
	font-family: Roboto;
	font-weight: 500;
	transition: all 0.3s ease;
}

QPushButton:pressed {
	background: #D4A017;
	color: #1A1A1A;
}</string>
    </property>
    <property name="text">
     <string>C A L C U L A T E</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="MinOrMax">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;
border: 2px solid #D4A017;
border-radius: 15px;
color: #FFD700;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="MinMaxInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1A1A1A;
border: 0px solid #D4A017;
border-radius: 25px;
color: #FFD700;
font-size: 18px;
font-family: Roboto;
padding-left: 10px;</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Поиск максимума</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Поиск минимума</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Method">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1A;
border: 2px solid #D4A017;
border-radius: 15px;
color: #FFD700;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="MethodInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1A1A1A;
border: 0px solid #D4A017;
border-radius: 25px;
color: #FFD700;
font-size: 14px;
font-family: Roboto;
padding-left: 10px;</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Метод половинчатого деления</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Метод золотого сечения</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Метод чисел Фиббоначи</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
